### PR TITLE
fix: minor demo fixes

### PR DESCRIPTION
**Pull request by 0xmad**

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -72,9 +72,9 @@ NEXT_PUBLIC_ROUND_ID="open-rpgf-1"
 NEXT_PUBLIC_TOKEN_NAME="OP"
 
 # How many votes each ballot can hold
-NEXT_PUBLIC_MAX_VOTES_TOTAL=30000000
+NEXT_PUBLIC_MAX_VOTES_TOTAL=150
 # How many votes each project can receive
-NEXT_PUBLIC_MAX_VOTES_PROJECT=5000000
+NEXT_PUBLIC_MAX_VOTES_PROJECT=50
```
Comment (ctrlc03) [inline .env.example:77]: what is this? like the max vote it can receive?

File: .env.example [hunk 2]
```diff
@@ -72,9 +72,9 @@ NEXT_PUBLIC_ROUND_ID="open-rpgf-1"
 NEXT_PUBLIC_TOKEN_NAME="OP"
 
 # How many votes each ballot can hold
-NEXT_PUBLIC_MAX_VOTES_TOTAL=30000000
+NEXT_PUBLIC_MAX_VOTES_TOTAL=150
```
[the remark this answers is not in the document]
Comment (0xmad) [reply .env.example:75]: No, it's not about it. It's how many vote weights you have per all the projects.

File: .env.example (hunk 1 above)
Comment (0xmad) [reply .env.example:77]: Right, it's max vote weight per project vote.

Comment (ctrlc03) [reply .env.example:77]: shouldn't we be taking it from the initial voice credit balance of the user which they received upon signing up? Also, one could put all their weight to 1 project, so the total and per project should be the same imo

Comment (0xmad) [reply .env.example:77]: I think yes, it's possible to do it.

Comment (0xmad) [reply .env.example:77]: Yes, let's merge this one because I need to add some changes for maci-cli to get the balance.

**Review by ctrlc03 — APPROVED** [review]

**Pull request merged**